website/docs: add notice for nginx ingress configuration requirement (#7027)
goauthentik/authentik: website/docs/providers/proxy/_nginx_ingress.md

Add these annotations to the ingress you want to protect

:::warning

> This configuration requires [allow-snippet-annotations](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#allow-snippet-annotations) to be enabled, for example by setting `controller.allowSnippetAnnotations` to `true` in your helm values for the ingress-nginx installation.

Reviewer: ```suggestion
This configuration requires that you enable [`allow-snippet-annotations`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#allow-snippet-annotations), for example by setting `controller.allowSnippetAnnotations` to `true` in your helm values for the ingress-nginx installation.
```

Reviewer: Avoiding the passive voice.
